src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -215,10 +215,18 @@
 
   .MessagingBubble--Message--Container--Own .MessagingAttachment--Container {
     margin-left: @size_5xl;
+    width: calc(100% - 4rem);
+    min-width: 13rem;
+    max-width: 21.125rem;
+    overflow: hidden;
```

Comment:
Is it that these four lines only apply to mobile messaging attachments within messaging bubles and that's why these are not in the `MessagingAttachment--Container` class on line 207?

Reply:
Ah I think you're right